docs/resources/file.md

```diff
@@ -0,0 +1,104 @@
+---
+subcategory: "Storage"
+---
+# databricks_file Resource
+
+This resource allows uploading and downloading files in [databricks_volume](volume.md) up to 2GB in octet-stream.
```

Comment:
Eventually this should support more than just uploading to volumes.
Should we update the doc now?